src/components/Layout.tsx

```diff
@@ -213,7 +213,7 @@ const Layout: React.FC<IProps> = ({
               <VisuallyHidden isHidden={isZenMode}>
                 <Footer />
               </VisuallyHidden>
-              <FeedbackWidget />
+              <FeedbackWidget location={path} />
```

Comment:
One common practice to deal with this is to do
```tsx
<FeedbackWidget key={path} />
```
that way, the component is going to remount everytime the path change and you will not have to do any manual reset in the inner component logic.